include/benchmark/benchmark.h

```diff
@@ -221,80 +287,54 @@ class State {
   //  BM_Compress   50         50   14115038  compress:27.3%
   //
   // REQUIRES: a benchmark has exited its KeepRunning loop.
-  void SetLabel(const std::string& label);
+  void SetLabel(const char* label);
```

Comment:
so there's State::SetLabel and benchmark::SetLabel?

that's a little confusing.